pkg/kubelet/util/util_darwin.go

```diff
@@ -0,0 +1,25 @@
+//go:build freebsd || linux || darwin
```

Comment:
wouldn't an `other` file that returns `false` is for everything but linux be clearer? this has a build tag for `linux` which is super confusing

is the filename trumping the build tags?

Reply:
maybe we should use the existing _unix / _windows / _unsupported files and just return false from _windows and _unsupported?